src/main/java/com/mathworks/ci/helper/MatlabBuilderConstants.java

```diff
@@ -36,6 +36,10 @@ public class MatlabBuilderConstants {
     public static final String JUNIT_RESULTS_CHX = "junitChecked";
     public static final String JUNIT_DEFAULT_FILE = "matlab-artifacts/test-reports/junit.xml";
 
+    public static final String HTML_FILE = "htmlTestReport";
+    public static final String HTML_RESULTS_CHX = "htmlTestReportChecked";
+    public static final String HTML_DEFAULT_FILE = "matlab-artifacts/test-reports/report.html";
```

Comment:
I don't think we specify html file, html test report generation behavior will mirror HTML Code coverage generation behavior. Refer this doc to understand : https://www.mathworks.com/help/matlab/ref/matlab.unittest.plugins.testreportplugin.producinghtml.html